packages/lesswrong/components/users/UsersProfile.jsx

```diff
@@ -7,7 +7,6 @@ import StarIcon from '@material-ui/icons/Star'
 import DescriptionIcon from '@material-ui/icons/Description'
 import MessageIcon from '@material-ui/icons/Message'
 import { withStyles } from '@material-ui/core/styles';
-import PropTypes from 'prop-types';
```

Comment:
Taking this import out is correct, but I'm confused: why didn't our usual eslint catch that it was unused?